pyclowder/connectors.py

```diff
@@ -690,7 +701,7 @@ def connect(self):
 
             self.channel.queue_bind(queue=self.rabbitmq_queue,
                                     exchange=self.rabbitmq_exchange,
-                                    routing_key="extractors." + self.extractor_name)
```

Comment:
lets get rid of the full exchange piece that is above here, we no longer want to support this.

Reply:
Was this already removed?